stable/mlrun-kit/templates/jupyter-notebook/pv.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.jupyterPersistencyStandalone.pvc.create -}}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: jupyter-pv
+  labels:
+    app.kubernetes.io/name: {{ template "mlrun-kit.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  storageClassName: "jupyter-host-path"
+  capacity:
+    storage: 2Gi
```

Comment:
This should be a value that can be configured in the values file

Reply:
this is the pv that create the local volume